docs/architecture/modernize-desktop/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Modernizing Desktop Apps on Windows 10 with .NET Core 3
+description: Learn how to modernize existing desktop apps with .NET Core 3.0
+author: nishanil
+ms.author: nanil
+ms.date: 04/14/2020
```

Comment:
@OliaG has been using September's date since this is when the book was originally written. It might make sense since some info is already outdated in this book because of that.

Reply:
Makes sense. Just updated.